crmsh/bootstrap.py

```diff
@@ -184,6 +187,18 @@ def _validate_nodes_option(self):
         for node in self.node_list:
             utils.ping_node(node)
 
+    def _validate_cluster_node(self):
+        """
+        Validate cluster_node on join side
+        """
+        if self.cluster_node and self.type == 'join':
+            try:
+                # self.cluster_node might be hostname or IP address
+                if socket.gethostbyaddr(self.cluster_node)[0] == utils.this_node():
```

Comment:
Should we compare ip addresses here? Reverse hostname lookup may not be reliable enough. For example, when multiple names resolve to a same ip in hosts file, reverse lookup returns only the 1st name.